Makefile

```diff
@@ -11,8 +11,60 @@ $(MAKEFILE):
 	git clone --quiet --depth 1 -b $(CI_BRANCH) $(CI_REPOSITORY) $(CI_PATH);
 -include $(MAKEFILE)
 
-protogen:
+OS := $(shell uname)
+PROTOC_DIR ?= $(HOME)/protoc
```

Comment:
I like the idea of fixing the protoc version. But if we are overriding any global installation, I think it may be better to install it in the current dir, in `./protoc`, or `./.protoc`. `$(HOME)/protoc` could conflict if you work in two src-d projects with different protoc versions. What do you think?

Reply:
thanks for the suggestion! 👍 

and there is `/protoc` in gitignore already (because $HOME == $CURRENT_DIR) in travis.